src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,114 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
 import './TodoList.scss';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
+interface Props {
+  todosFromServer: Todo[];
+  selectUser: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const TodoList: React.FC<Props>
+  = ({ todosFromServer, selectUser }) => {
+    const [filter, setFilter] = useState<string>('');
+    const [selectFilter, setSelectFilter] = useState('all');
+
+    const filteredByState
+      = todosFromServer.filter(todo => {
+        if (todo.completed === true && selectFilter === 'completed') {
+          return true;
+        }
+
+        if (todo.completed === false && selectFilter === 'active') {
+          return true;
+        }
```

Comment:
i belive you can remove `todo.completed === true` and `todo.completed === false`
also, use switch case here

Reply:
![image](https://user-images.githubusercontent.com/63620508/174676825-1e277fae-01ae-4da1-87a9-0ffa499db5a7.png)

No unfortunately I can't remove this because without this logic it will show all tasks no matter which option is currently selected.

Reply:
And in this case I'm not quite sure how to switch case here